app/src/main/java/io/pumpkinz/pumpkinreader/NewsDetailFragment.java

```diff
@@ -37,7 +39,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
-        News news = Parcels.unwrap(getActivity().getIntent().getParcelableExtra(Constants.NEWS));
+        final News news = Parcels.unwrap(getActivity().getIntent().getParcelableExtra(Constants.NEWS));
```

Comment:
Because it was going to be passed to the anonymous class, which requires any outside variable to be final.
Now that I've moved the anonymous function to setUpFAB, I will make this not final.